test/run

```diff
@@ -68,10 +71,13 @@ prepare() {
 }
 
 run_test_application() {
-    if [[ -z $2 ]]; then
-        docker run --user=100001 --rm --cidfile=${cid_file} $1 ${IMAGE_NAME}-testapp
+    if [[ $1 == app ]]; then
+        docker run --user=100001 --rm --cidfile=${cid_file} $2 ${IMAGE_NAME}-testapp
```

Comment:
Ah sorry, I should have been more specific. What I meant was something along the lines of:

```docker run --user=100001 --rm --cidfile=${cid_file} $2 ${IMAGE_NAME}-test${1}```

so that we do not need any if statements at all.

Reply:
Actually I wanted to handle cases when an invalid argument was passed to the function.

Reply:
Right, I missed that `set -e` is not used in nodejs test script. In that case it makes sense to me.